pyvo/registry/regtap.py

```diff
@@ -128,6 +131,12 @@ def search(*constraints:rtcons.Constraint, includeaux=False, **kwargs):
         This may result in duplicate capabilities being returned,
         especially if the servicetype is not specified.
 
+    maxrec : int
+        Overrides the RegTAP server's default limit on the number of rows to
+        return.  You may need to use this if you want to retrieve more
```

Comment:
This is a bit confusing. The DALI standard states that `The service may also enforce a limit on the value of MAXREC that is smaller than the value in the request.` so I don't think one can use this to override server's `MAXREC`.